pkg/operator/metrics.go

```diff
@@ -23,9 +23,9 @@ var (
 		[]string{checkNameLabel},
 	)
 
-	clusterCheckErrrorMetric = metrics.NewCounterVec(
-		&metrics.CounterOpts{
-			Name:           "vsphere_cluster_check_errors_total",
+	clusterCheckErrrorMetric = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Name:           "vsphere_cluster_check_errors",
 			Help:           "Number of failed vSphere cluster-level checks performed by vsphere-problem-detector.",
```

Comment:
You should update the help text, it's not nr. of failed tests any longer.
\+ the same for vsphere_node_check_errors_total below.

Reply:
done.